src/main/kotlin/com/exactpro/th2/http/client/Main.kt

```diff
@@ -234,4 +238,21 @@ private inline fun <reified T> load(defaultImpl: Class<out T>): T {
 
 private fun createSequence(): () -> Long = Instant.now().run {
     AtomicLong(epochSecond * SECONDS.toNanos(1) + nano)
-}::incrementAndGet
\ No newline at end of file
+}::incrementAndGet
+
+private fun MessageRouter<EventBatch>.storeEvent(name: String, eventId: String, messagesId: List<MessageID>? = null) : String {
```

Comment:
> messagesId: List<MessageID>? = null

Type can be replaced by `MessageID?` since we always pass a single list element to it